store/src/java/com/zimbra/cs/mailbox/MailSender.java

```diff
@@ -1402,4 +1408,33 @@ public static void unregisterPreSendMailListener(PreSendMailListener listener) {
             }
         }
     }
+    
+    /**
+     * Setup JavaMail session properties for STARTTLS based the ldap configuration
+     * @param account
+     * @param smtpSession
+     * @throws ServiceException
+     */
+    public static void setupStartTlsMode(Account account, Session smtpSession) throws ServiceException {
+        SmtpStartTlsMode startTlsMode = Provisioning.getInstance().getDomain(account).getSmtpStartTlsMode();
+        if (startTlsMode == null) {
+        		startTlsMode = Provisioning.getInstance().getLocalServer().getSmtpStartTlsMode();
```

Comment:
nit: (more tabs...) it appears indentation is off here (and below), I'd expect 4 spaces past the start of **if** block